examples/github.com-dakanji-RefindPlus/Makefile

```diff
@@ -1,6 +1,12 @@
 all:
+	make clean
+	# Uncomment the commented lines below to rebuild the Docker image
+	#cp ../../build-edk2.sh . && cp ../../clone-edk2.sh . && cp ../../entry.sh .
+	#EDK2_VERSION=RefindPlusUDK DOCKERFILE_PATH=../../Dockerfile.RefindPlus IMAGE_NAME=xaionaro2/edk2-builder:RefindPlusUDK ../../hooks/build
+	# Comment out the docker pull line in Build.sh file
```

Comment:
Let's remove this comment. I can try to fix the main `Dockerfile` if you could let me know what is wrong with it :)

Reply:
Yes sure. I was just trying to automate this:
```
EDK2_VERSION=<THE SPECIAL TAG HERE> DOCKERFILE_PATH=Dockerfile IMAGE_NAME=<SOME IMAGE NAME>:<SOME TAG> hooks/build
```
It will build a docker image `<SOME IMAGE NAME>:<SOME TAG>` using the custom EDK2 repository.
```

Reply:
done. main Dockerfile still does not build for
https://github.com/startergo/edk2-builder-docker/tree/main/examples/github.com-joevt-RefindPlus. It never did.

Reply:
CC:@xaionaro I found that the error is produced by the latest entry.sh. If I use this one:
[entry.sh](https://raw.githubusercontent.com/startergo/edk2-builder-docker/58db7e3c8901767cb4e9d2c076dc66db81a46af5/examples/sourceforge.net-srs5694-Refind/entry.sh)
it works fine. Don't know what is causing the error. Please take a close look.